amy.txt

```diff
@@ -0,0 +1,2 @@
+Amna is very brilliant java student. Mas haa Allah :-)
+Amna belives that appreciation by java tutor(Usman Saeed) is a key to success.Yup :-)
```

Comment:
Only appreciation is nothing.

Reply:
Yes I totally agree with you. Likin sir apka hr dia task itni mihnat k bd achieve kia jata. Is liye thori si appreciation tou banti hai in sb k bd :-)
And sir friendly raha karye , daraya na karye har time master branch ka kh kr :-)